pkg/beyla/config.go

```diff
@@ -131,12 +134,14 @@ type Config struct {
 
 	Attributes Attributes `yaml:"attributes"`
 	// Routes is an optional node. If not set, data will be directly forwarded to exporters.
-	Routes       *transform.RoutesConfig       `yaml:"routes"`
-	NameResolver *transform.NameResolverConfig `yaml:"name_resolver"`
-	Metrics      otel.MetricsConfig            `yaml:"otel_metrics_export"`
-	Traces       otel.TracesConfig             `yaml:"otel_traces_export"`
-	Prometheus   prom.PrometheusConfig         `yaml:"prometheus_export"`
-	Printer      debug.PrintEnabled            `yaml:"print_traces" env:"BEYLA_PRINT_TRACES"`
+	Routes            *transform.RoutesConfig       `yaml:"routes"`
+	NameResolver      *transform.NameResolverConfig `yaml:"name_resolver"`
+	Metrics           otel.MetricsConfig            `yaml:"otel_metrics_export"`
+	Traces            otel.TracesConfig             `yaml:"otel_traces_export"`
+	Prometheus        prom.PrometheusConfig         `yaml:"prometheus_export"`
+	Printer           debug.PrintEnabled            `yaml:"print_traces" env:"BEYLA_PRINT_TRACES"`
+	JSONPrinter       json.PrintEnabled             `yaml:"print_traces_json" env:"BEYLA_PRINT_TRACES_JSON"`
+	JSONPrinterIndent json.PrintIndent              `yaml:"traces_json_indent" env:"BEYLA_TRACES_JSON_INDENT"`
```

Comment:
Maybe rename these as `print_traces_json_indent` and `BEYLA_PRINT_TRACES_JSON_INDENT`?

Reply:
Done